### ISSUE TITLE
Question: Unifi Protect media in Gallery

### ISSUE THREAD

**Issue by wvdeadhead** [TarheelGrad1998/gallery-card#43]
Are you aware of a way to integrate the media from the Unifi Protect integration into the card?  I would love to use this for my Unifi cameras and have an integration similar to https://github.com/dermotduffy/frigate-hass-card.

I've tried multiple media-source paths but can't quite seem to figure it out.

**Comment by TarheelGrad1998** [comment]
I'm not familiar with Unifi Protect explicitly.  Are the videos somewhere you can add a symbolic link or similar?

This is what I have done for Synology Surveillance Station:
First, added the volume link into the docker container
Then add that volume as a media source in confinguration.yaml:

```
  media_dirs:
    surveillance: /mnt/media/surveillance
```

Then add that media source as an entity on the card:
```
entities:
  - path: media-source://media_source/surveillance/
    recursive: true
```

I have on my to-do list adding other media source, such as from the media browser, but haven't gotten around to that yet.

**Comment by wvdeadhead** [comment]
Thanks @TarheelGrad1998, I was trying to use the built in media sources paths from the HASS gui.  I'll see if there's a way I can map the directory in the config like you mentioned.

Nice project btw.  I appreciate people like you helping out the community.

**Comment by stefanschaedeli** [comment]
> Thanks @TarheelGrad1998, I was trying to use the built in media sources paths from the HASS gui. I'll see if there's a way I can map the directory in the config like you mentioned.

I fully agree, would be nice if this galary card could display the content of the media browser (where other integrations did all the work already to make it available :)
